src/styles/blocks/form.scss

```diff
@@ -0,0 +1,57 @@
+.form {
+  cursor: pointer;
+  max-width: 411px;
+
+  @include onDesktop {
+    flex-direction: row;
+    justify-content: space-between;
+    column-gap: 24px;
+  }
+  &__input {
+    @include InputStyle;
+    margin-bottom: 16px;
+    height: 48px;
+  }
+
+  &__text-area {
+    @include InputStyle;
+    margin-bottom: 46px;
+    resize: none;
+    height: 148px;
+  }
+}
+
+.button {
+  display: flex;
+  align-items: center;
+  justify-self: center;
+  justify-content: center;
+  border-radius: 8px;
+  width: 100%;
+  height: 56px;
+  text-decoration: none;
+  box-sizing: border-box;
+  margin-bottom: 96px;
+  background-color: $main-text-color;
+  color: $element-white;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 27px;
+  text-align: center;
+  transition: all 300ms;

```

Comment:
It is better to indicate what exactly this `transition` should work for, and not to set it for `all`.

Fix this in all code.